felix/dataplane/external/ext_dataplane.go

```diff
@@ -199,6 +199,10 @@ func (fc *extDataplaneConn) SendMessage(msg interface{}) error {
 		envelope.Payload = &proto.ToDataplane_GlobalBgpConfigUpdate{GlobalBgpConfigUpdate: msg}
 	case *proto.Encapsulation:
 		envelope.Payload = &proto.ToDataplane_Encapsulation{Encapsulation: msg}
+	case *proto.ServiceUpdate:
+		envelope.Payload = &proto.ToDataplane_ServiceUpdate{ServiceUpdate: msg}
+	case *proto.ServiceRemove:
+		envelope.Payload = &proto.ToDataplane_ServiceRemove{ServiceRemove: msg}
 
 	default:
 		log.WithField("msg", msg).Panic("Unknown message type")
```

Comment:
@AloysAugustin should we downgrade this to a warning, or, even better(!?) factor out an "WrapPayloadWithEnvelope()" function that the calc_graph_fv_tests can call on every output object to check it is handled?

Reply:
Good point! I'll see if I can figure something out something like this to prevent similar issues in the future.